### PR TITLE
Feature/175symptomscheckstyling

### PR DESCRIPTION
**Pull request by agmaal1**
**Quick Description of the PR**
Updated styling for Symptoms Check for Desktop
https://feature175symptomscheckstyling.fightpandemics.xyz/symptoms-check#step1

**An Overview of the Changes**
Updated styling for Symptoms Check in a separate file without modifying step wizard's Components 
client/src/components/SymptomsCheck/SymptomsCheckStyles.js.

Main Changes in:
client/src/pages/SymptomsCheck.js (modified)
client/src/components/SymptomsCheck/SymptomsCheckStyles.js (created)

minor changes in:
client/src/components/StepWizard/WizardNav.js
client/src/templates/layouts/Main.js
client/src/GlobalStyles.js

THE REST ARE  ESLINT RELATED

### CODE REVIEW

**Comment by MohamedEl1** [comment]
Hi @agmaal1 there seems to be some conflicting files

**Comment by snowiesuet** [comment]
80% of the changes are eslint related, its hard to review the actual changes..

File: client/src/components/StepWizard/WizardContainer.js [hunk 1]
```diff
@@ -1,25 +1,69 @@
...
+
+  @media screen and (min-width: ${tablet.narrow.minWidth}) {
+    display: flex;
+    justify-content: center;
     margin: ${marginTop} auto;
-    width: 100%;
-    max-width: 650px;
+    max-width: 850px;
+
+    .title{
```
Comment (snowiesuet) [inline client/src/components/StepWizard/WizardContainer.js]: We have <Heading> & <TextLabel> components so you don't have to create new styles for them

Comment (agmaal1) [reply client/src/components/StepWizard/WizardContainer.js]: got rid of that. please take a look to the 
client/src/components/SymptomsCheck/SymptomsCheckStyles.js

**Comment by agmaal1** [comment]
@Annie @robinv85 @snowiesuet  I’ve been styling the checker overwriting the step wizard components but those are not built on top of ant components but they have their own styles, that leads to more custom components instead of reusable ones. 
https://feature175symptomscheckstyling.fightpandemics.xyz/symptoms-check#step1
Shouldn't we modify the step wizard's components first to make them reusable. Otherwise this is going to happen in the need help checker as well.

**Comment by robinv85** [comment]
@agmaal1 sure, the point of the StepWizard components was to make them reusable, requirements have changed and it's been a while since I've looked at the code but if you think they need to change to be more reusable for the requirements then this seems like the way to go.

**Comment by snowiesuet** [comment]
I think if there are no big issues and styling looks fine we should merge it for now so it can be merged with other changes. @anniegiang what do you think?

**Pull request merged**